lib/ansible/modules/cloud/amazon/dynamodb_table.py

```diff
@@ -30,6 +30,7 @@
 author: Alan Loi (@loia)
 requirements:
   - "boto >= 2.37.0"
+  - "boto3 >= 1.4.4 (for tagging)"
```

Comment:
DynamoDB tagging was not supported by AWS until recently. boto3 1.4.4 was the first version to support tagging.